packages/modal/src/Modal.styled.js

```diff
@@ -36,24 +36,30 @@ export const ConstrainedCard = styled(Card)`
   position: relative;
   display: flex;
   flex-direction: column;
+  border: none;
+  overflow: hidden;
 `;
 
 export const ContentWrapper = styled.div`
   flex: 1;
   overflow: auto;
   -webkit-overflow-scrolling: touch;
+  padding: 16px;
 `;
 
-export const Header = styled.div`
+export const HeaderWrapper = styled.header`
+  overflow: auto;
```

Comment:
The header could have a default style to avoid code duplication and inconsistencies.
![image](https://user-images.githubusercontent.com/1459283/61217100-60025d00-a6e5-11e9-923a-c189fa825a80.png)

You can follow the specs from the same design you got the button style:
![image](https://user-images.githubusercontent.com/1459283/61217264-af488d80-a6e5-11e9-8db5-04bc1a21b9d3.png)

Reply:
I don't know if we want this, to prevent duplication and easy maintenance we need to use the farmblocks `Text` to display this title. I have suggested a `title` prop to display those titles, but @alcferreira prefer a `header` element to be fully customizable. Maybe we can have both, what do you guys think?

Reply:
We can export a secondary component `ModalTitle` that extends a `Text` with the proper font size and weight.

It would be used like `<Modal header={<ModalTitle>Hello</ModalTitle>} ... />`